x/auth/tx/service_test.go

```diff
@@ -139,14 +154,23 @@ func (s IntegrationTestSuite) TestGetTxEvents() {
 	s.Require().Equal(len(grpcRes.Txs), 1)
 	s.Require().Equal("foobar", grpcRes.Txs[0].Body.Memo)
 
+	var getTxRes tx.GetTxsEventResponse
+
+	// Query the tx via grpc-gateway empty params.
+	restRes, err := rest.GetRequest(fmt.Sprintf("%s/cosmos/tx/v1beta1/txs", val.APIAddress))
+	s.Require().Error(val.ClientCtx.JSONMarshaler.UnmarshalJSON(restRes, &getTxRes))
+
+	// Query the tx via grpc-gateway without pagination.
+	_, err = rest.GetRequest(fmt.Sprintf("%s/cosmos/tx/v1beta1/txs?event=%s", val.APIAddress, "message.action=send"))
+	s.Require().NoError(err)
+
 	// Query the tx via grpc-gateway.
-	restRes, err := rest.GetRequest(fmt.Sprintf("%s/cosmos/tx/v1beta1/txs?event=%s&pagination.offset=%d&pagination.limit=%d", val.APIAddress, "message.action=send", 0, 1))
+	restRes, err = rest.GetRequest(fmt.Sprintf("%s/cosmos/tx/v1beta1/txs?event=%s&pagination.offset=%d&pagination.limit=%d", val.APIAddress, "message.action%3Dsend", 0, 1))
 	s.Require().NoError(err)
-	var getTxRes tx.GetTxsEventResponse
 	s.Require().NoError(val.ClientCtx.JSONMarshaler.UnmarshalJSON(restRes, &getTxRes))
-	s.Require().Equal(len(grpcRes.Txs), 1)
+	s.Require().Equal(len(getTxRes.Txs), 1)
 	s.Require().Equal("foobar", getTxRes.Txs[0].Body.Memo)
-	s.Require().NotZero(grpcRes.TxResponses[0].Height)
+	s.Require().NotZero(getTxRes.TxResponses[0].Height)
```

Comment:
Could we switch to table tests here? (at least the gRPC-gateway ones, they seems similar)